extensions/amp-form/0.1/amp-form.js

```diff
@@ -229,12 +242,22 @@ export class AmpForm {
     this.form_.addEventListener(
         'submit', this.handleSubmitEvent_.bind(this), true);
     this.form_.addEventListener('blur', e => {
-      onInputInteraction_(e);
+      checkUserValidityAfterInteraction_(dev().assertElement(e.target));
       this.validator_.onBlur(e);
     }, true);
+    this.form_.addEventListener('change', e => {
+      const input = dev().assertElement(e.target);
+      this.verifier_.onCommit(input, updatedElements => {
+        updatedElements.forEach(updatedElement => {
+          checkUserValidityAfterInteraction_(updatedElement);
+        });
+        this.validator_.onBlur(e);
```

Comment:
This function's name is not very clear. It's part of the `amp-form` `FormValidator` API and it hides any displayed messages. Originally it was always called on blur in the form, but that behavior is also needed when the verifier calls `afterVerify`.